abel/tools/vmi.py

```diff
@@ -60,6 +185,8 @@ def angular_integration(IM, origin=None, Jacobian=True, dr=1, dt=None):
         integrated intensity array (vs radius).
 
     """
+    _deprecate('angular_integration() is deprecated, '
```

Comment:
Why not suggest angular_integration_3D or angular_integration_3D specifically here?

Reply:
Done!